config/exec.go

```diff
@@ -41,7 +41,7 @@ type ExecConfig struct {
 	Env *EnvConfig `mapstructure:"env"`
 
 	// KillSignal is the signal to send to the command to kill it gracefully. The
-	// default value is "SIGTERM".
+	// default value is "SIGINT".
```

Comment:
Given that this comment was wrong and yet only 30 lines of code below the definition above (line 13), perhaps nuke the comment outright?

Reply:
Agreed.